static/js/publisher/release/components/revisionsListRow.js

```diff
@@ -114,10 +184,74 @@ const RevisionsListRow = props => {
       {showProgressiveReleases && (
         <td>
           {progressiveState && (
-            <ProgressiveBar
-              percentage={progressiveState.percentage}
-              readonly={true}
-            />
+            <div
```

Comment:
It would be good to extract it to separate component to contain its functionality outside of the row.